packages/core/src/runtime/stylis/globalPlugin.ts

```diff
@@ -0,0 +1,55 @@
+import { RULESET, tokenize } from 'stylis';
+import type { Middleware } from 'stylis';
+
+export const globalPlugin: Middleware = element => {
+  switch (element.type) {
+    case RULESET:
+      if (typeof element.props === 'string') {
+        throw new Error(
```

Comment:
Why would throwing an error in production with an empty message be a good thing ?

Reply:
Save bundle size a bit. I will change it

Reply:
I'm also wondering why throw the error in production at all ? it seems inconsistent with other errors we have which only throw at devtime

Reply:
> I'm also wondering why throw the error in production at all ? it seems inconsistent with other errors we have which only throw at devtime

It's consistent: we _throw_ always, but we _don't log_ in production...

Reply:
Okay, except this one:
https://github.com/microsoft/griffel/blob/ebe44fdd9c34a87a4106c95c5b47f5719c215e8f/packages/react/src/makeStyles.ts#L32-L41

But I don't see a reasonable way to not throw there.

For example there, we can do:

```diff
      if (typeof element.props === 'string') {
+        if (process.env.NODE_ENV !== 'production') {
        throw new Error(
          `"element.props" has type "string" (${JSON.stringify(
            element.props,
            null,
            2,
          )}), it's not expected. Please report a bug if it happens.`,
        );
+     }
+     return;
      }
```

But what to do for another one instead of throwing? Ideas?